modules/service/src/main/scala/lucuma/odb/graphql/mapping/ObservationMapping.scala

```diff
@@ -117,4 +119,18 @@ trait ObservationMapping[F[_]]
     effectHandler(readEnv, calculate)
   }
 
+  def validationsQueryHandler: EffectHandler[F] = {
```

Comment:
I think having validations be part of the observation is very nice from a usability standpoint. The only thing I don't like is that we need to look up CfP information for each of the observations.